src/app.py

```diff
@@ -17,7 +17,7 @@
 import redis
 from celery.app import Celery
 
-from openrelik_worker_common.debug_utils import start_debugger
+from .openrelik_worker_common.debug_utils import start_debugger
```

Comment:
?

Reply:
Importing with a starting dot means it will first search for a local import and go for the PYTHONPATH if it can't find it. This makes it easier to develop both a worker and the openrelik_worker_common lib at the same time. You just place a copy of the openrelik_worker_common in the ./src folder and uncomment the library in the pyproject.toml.